scripts/SPM/Sources/Core/File+Reset.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  File+Reset.swift
+//  Core
+//
+//  Created by Ellen Shapiro on 4/8/19.
+//
+
+import Foundation
+import Files
+import ShellOut
+
+extension File {
+    
+    func resetToGitHEAD() throws {
+        let cmd = ShellOutCommand(string: "git checkout HEAD -- \"\(self.path)\"")
```

Comment:
There might be a problem when you are in a detached HEAD state in git (e.g checking out a tag).  See the circle ci workflows

Reply:
Also will this revert any changes made to the project file when building using fastlane? Our `tfbeta` lane updates the build number.

Reply:
It will only check out `HEAD` for the single file represented by the `file` object. It will not affect other files.